pypif/obj/common/property.py

```diff
@@ -45,8 +46,10 @@ def __init__(self, name=None, scalars=None, vectors=None, matrices=None, files=N
         self.licenses = licenses
         self._conditions = None
         self.conditions = conditions
-        self._method = None
-        self.method = method
+        self._methods = None
+        self.methods = methods
+        if 'method' in kwargs:
+            self.methods = method
```

Comment:
@joannehill Let's just swap the order here (51, 51) and (49, 50). Then I think this will be in good shape.